homeassistant/components/spotify/const.py

```diff
@@ -6,6 +6,11 @@
 DATA_SPOTIFY_ME = "spotify_me"
 DATA_SPOTIFY_SESSION = "spotify_session"
 
+# Stores the devices retrievd from Spotify
+# Used by custom components so another spotify auth
+# won't be required
```

Comment:
But you will still need auth tokens from web player, right? As I remember only web player can start cast.

Reply:
Yes. Still need the powerful token for spotcast to start the Spotify app on a cast device